src/cli/init.ts

```diff
@@ -49,10 +54,16 @@ export default class InitCommand extends BaseCommand {
     if (!isAddress(offerId)) throw new Error('Invalid Offer Address')
 
     if (fs.existsSync(this.dbPath as string)) {
-      if (!this.parsedArgs.flags.skipPrompt && !(await this.confirm('Are you sure you want to overwrite your current DB? All data will be erased! (y/n)'))) {
+      const overrideDb = this.parsedArgs.flags['override-db']
+
+      if (overrideDb === false) {
```

Comment:
Haven't checked the rest of the code to see if this is a common idiom, but usually comparing booleans is weird. Shouldn't this be changed to `!overrideDb`?

Reply:
Nope, `undefined` is a valid value here, which is in JS otherwise considered a "false-ish"